Changed the class name ayumi=>table and deleted blank lines

ayumi-ayumi opened the pull request:
I changed the class name "ayumi" to "table" and deleted blank lines on html.

artkontraer commented:
> I changed the class name "ayumi" to "table" and deleted blank lines on html. 

I don't think that works, because my class is already called table.

ayumi-ayumi commented:
> > I changed the class name "ayumi" to "table" and deleted blank lines on html.
> 
> I don't think that works, because my class is already called table.

I changed the class name whichever is called "ayumi" into "table" (on table, th, tb tags)
